benchmark/go/mnist.go

```diff
@@ -0,0 +1,83 @@
+package main
```

Comment:
This file is reduplicated with https://github.com/wangkuiyi/gotorch/blob/develop/data_test.go , I will polish it in another PR.